### DISCUSSION TITLE
Welcome to django-organizations Discussions!

### DISCUSSION THREAD

**Discussion by bennylope** [bennylope/django-organizations#211] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by juuisle** [comment]
So, this is the first comment. Thank you for building this package. I am just a user, I do nothing, but I would like to contribute if I can.
